### DISCUSSION TITLE
Add DAX Code to Measure Descriptions

### DISCUSSION THREAD

**Discussion by lsener** [TabularEditor/TabularEditor3#1345] [🙋♀️TE3 Q&A]
Newby here. Did this in TE2, where to add into TE3? 

Found this code for adding into TE2 and works fabulously to add the DAX code into the description field for measures and another slightly different code for the calculated columns. I use it every day and due to a new computer I have to reload this. The new version of TE2 doesn't seem to have this is the same place as my version (2.16.6) and I've just gotten TE3 and haven't figured out my way around yet. 

Can someone tell me in TE2 (2.25.0) or TE3 (3.16.2) where I add this to make it work since it's not the same anymore? TE3 probably would make the most sense so I'll start to use it. But either way....I love this option. I just don't know enough of how to use TE3 yet.  

**For Measures:** 
![image](https://github.com/TabularEditor/TabularEditor3/assets/7001059/cf0ae614-208f-471d-b51d-ab0c83cf076e)

Goes here as a Custom Sample Script. (At least how the video I used told me how to use it. Brunner BI 2021, https://en.brunner.bi/post/time-saving-scripts-in-tabular-editor-3):
![image](https://github.com/TabularEditor/TabularEditor3/assets/7001059/6fac8013-0902-446e-8b0d-ee19c91a76eb)
 
**For Calculated Columns:** 
![image](https://github.com/TabularEditor/TabularEditor3/assets/7001059/79e39186-e68f-4323-9c4c-1c90de4f9ba6)

**Comment by lsener** [comment]
Poster Linda here....I figured it out. It wanted the code first then the Green plus button to save. I thought it was the other way around. The New Tab is called C# Script instead of Advanced Scripting. But otherwise works the same. Should have tried that more before posting. But maybe someone will see this and find it useful. It's great to be able to see the code especially if you are working with a share dataset since you don't get the see the code behind the measures, which is what I had. Any other comments welcome or suggestions for the newby. I haven't done the training, but haven't found a lot of videos for individual learning of tasks either. Still feeling my way around.

**Reply by otykier** [reply]
Glad to hear you figured it out :-)

In case you’re not already aware, we have a very comprehensive self-paced training section on https://learn.tabulareditor.com - lots of content and there’s even an exam to test your skills. And the best part: It’s all free!
